src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -221,7 +223,10 @@ private IEnumerable<GeometryObject> Tessellate(Curve curve)
             {
                 var xyz0 = new XYZ(verts[i] * conv, verts[i + 1] * conv, verts[i + 2] * conv);
                 var xyz1 = new XYZ(verts[i + 3] * conv, verts[i + 4] * conv, verts[i + 5] * conv);
-
+                if (xyz0.DistanceTo(xyz1) < DocumentManager.Instance.CurrentUIApplication.Application.ShortCurveTolerance)
+                {
+                    continue;
+                }
```

Comment:
Yes, you are correct. The line will disappear. I considered this and figured that this was an acceptable loss. We are talking about lines that are shorter than 0.0013 units in Revit. `Polyline.Create` is a good idea, but I wonder if it is subject to the same limitations. That is, what if someone passes in a tiny little circle?